Jenkinsfile

```diff
@@ -62,35 +62,30 @@ pipeline {
               }
 
               withTools("${env.TOOLS_VERSION}") {
-                // Fetch all dependencies using XCommon CMake
-                sh "cmake -G 'Unix Makefiles' -B build"
-
+                sh "git submodule update --init"
+                createVenv("requirements.txt")
+                withVenv() {
+                  sh "pip install -r requirements.txt"
+                  // Check that the app_configs_autogen.yml file is up to date
+                  sh "python tests/tools/app_configs_autogen/collect_configs.py check"
+                }
                 // Build the loopback version of the configs for 316 and rename them to have _i2sloopback
-                sh 'xmake -C app_usb_aud_xk_316_mc -j16 PARTIAL_TEST_CONFIGS=1 TEST_SUPPORT_CONFIGS=1 EXTRA_BUILD_FLAGS=-DI2S_LOOPBACK=1'
+                sh "cmake -S app_usb_aud_xk_316_mc/ -B build -DPARTIAL_TESTED_CONFIGS=1 -DTEST_SUPPORT_CONFIGS=1 -DEXTRA_BUILD_FLAGS='-DI2S_LOOPBACK=1'"
```

Comment:
Since the winbuiltin config has moved to partial_test.cmake, I think you can remove `-DTEST_SUPPORT_CONFIGS=1` here to avoid unnecessarily building the DFU upgrade XEs with I2S_LOOPBACK enabled.